var/spack/repos/builtin/packages/dislin/package.py

```diff
@@ -0,0 +1,54 @@
+##############################################################################
+# Copyright (c) 2013-2017, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/spack/spack
+# Please also see the NOTICE and LICENSE files for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+
+
+class Dislin(Package):
+    """DISLIN is a high level and easy to use graphics library for displaying
+       data as curves,  bar graphs,  pie charts,  3D-colour plots,  surfaces,
+       contours and maps. The software is available for several C, Fortran 77
+       and Fortran 90 compilers."""
+
+    homepage = "http://www.mps.mpg.de/dislin"
+    url      = "ftp://ftp.gwdg.de/pub/grafik/dislin/linux/i586_64/dislin-11.0.linux.i586_64.tar.gz"
+
+    version('11.0.linux.i586_64', '6fb099b54f41db009cafc702eebb5bc6')
+
+    depends_on('motif')
+    depends_on('mesa')
+
+    def setup_environment(self, spack_env, run_env):
+        spack_env.set('DISLIN', self.prefix)
+        run_env.set('DISLIN', self.prefix)
+        run_env.prepend_path('PATH', self.prefix)
+        run_env.prepend_path('LD_LIBRARY_PATH', self.prefix)
+        run_env.prepend_path('LD_LIBRARY_PATH', self.spec['motif'].prefix.lib)
+        run_env.prepend_path('LD_LIBRARY_PATH', self.spec['mesa'].prefix.lib)
+
+    def install(self, spec, prefix):
+        INSTALL = Executable('./INSTALL')
+        INSTALL()
+        with working_dir('examples'):
+            install('dislin_d.h', prefix)
```

Comment:
Are we installing a header directly into the `prefix`?

Reply:
Most header files are copied to prefix by 'INSTALL' provided by this package, but 'dislin_d.h' is not copied to the prefix, while a dependent need this header file and others that are already in the prefix, so I also moved this one to the prefix.

Reply:
To be clear, my question was to be sure that you didn't mean:
```python
with working_dir('examples'):
    install('dislin_d.h', prefix.include)
```

Reply:
No, the dislin header files are installed to the prefix rather than prefix.include (which is what 'INSTALL' does). It is a bit strange, but I don't want to move around all these files, so I just put 'dislin_d.h' to the prefix to be consistent with 'INSTALL'.